exporter/loggingexporter/logging_exporter.go

```diff
@@ -37,8 +37,6 @@ func NewTraceExporter(config configmodels.Exporter, logger *zap.Logger) (exporte
 			// TODO: Add ability to record the received data
 			return 0, nil
 		},
-		exporterhelper.WithTracing(true),
-		exporterhelper.WithMetrics(true),
```

Comment:
Just to clarify, this does not disable tracing/metrics functionality, does it? Is it now done elsewhere or is no longer an option that needs to be passed to NewTraceExporter?

Reply:
Correct, tracing/metrics are enabled. Disabling those becomes a function of the collector itself (via an extension for z-pages and command-line for metrics).